change/@ni-nimble-angular-91d3afae-1d79-413d-9ca3-c0cd87b52ec7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Note: We should wait to merge this PR until we're ready to uptake the breaking change in SLE.

Reply:
Scanned the discussions from the spec update and don't think it came up but would we avoid a breaking change if we introduced `text-visible` instead of `text-hidden`?
If our usage guidelines are generally that the text should be hidden in an icon column and only shown for special cases (ambiguous icons or multiple mappings to the same icon) then maybe it would encourage good defaults and prevent a breaking change (I'm not sure what our usage guidance will be).

Reply:
The breaking change didn't end up being a problem to uptake. Angular's strict template checking didn't mind having `text-hidden` on the mapping elements even though the directive change hasn't been merged yet. (If it were a non-boolean value where I had to set `text-hidden="foo"`, then it would've been a problem.)

With that knowledge, are you still in favor of potentially changing from `text-hidden` to `text-visible`? Should I try to get more clarity on our usage guidance to influence this discussion?

Reply:
I like good defaults and my hunch is that would be a better default but wouldn't hold anything up for it.

Reply:
Actually maybe not even sure about my hunch, might not even be worth the effort to rename.

Reply:
@fredvisser - Do you have an idea of what the best practices will be for putting text next to the icons? Do you think the recommendations will make it more common to have text in the column with the icon or to have just the icon/spinner?

Reply:
Yeah, our current plan is to default to `icon + text` for most columns. After standardizing on the icons and text we use across SLE apps, we should have a solid majority of icon columns configured this way.

Reply:
Thanks. I'll leave this with `text-hidden` as currently implemented.